docs/stdlib/numbers.rst

```diff
@@ -607,7 +626,7 @@ from :eql:type:`str` and :eql:type:`json`.
                   std::bit_xor(l: int32, r: int32) -> int32
                   std::bit_xor(l: int64, r: int64) -> int64
 
-    Bitwise exclusive OR operator for 2 intergers.
+    Returns the result of an exclusive, bitwise OR operator for two integers:
```

Comment:
```suggestion
    Returns the result of an exclusive bitwise OR operation for two integers.
```

The comma may be grammatically correct (not sure), but after surveying some other documentation, the convention seems to be to omit the comma when referring to exclusive bitwise operations.